cluster/gce/gci/configure.sh

```diff
@@ -668,14 +608,12 @@ function install-kube-binary-config {
     else
       cp "${src_dir}/kube-apiserver.tar" "${dst_dir}"
       cp "${src_dir}/kube-controller-manager.tar" "${dst_dir}"
-      cp "${src_dir}/cloud-controller-manager.tar" "${dst_dir}"
       cp "${src_dir}/kube-scheduler.tar" "${dst_dir}"
       cp -r "${KUBE_HOME}/kubernetes/addons" "${dst_dir}"
     fi
-    load-docker-images
+    log-wrap "LoadDockerImages" load-docker-images
     mv "${src_dir}/kubelet" "${KUBE_BIN}"
     mv "${src_dir}/kubectl" "${KUBE_BIN}"
-    mv "${src_dir}/auth-provider-gcp" "${KUBE_BIN}"
```

Comment:
@mikedanese What's the status of gke-auth-provider-gcp? I see that it's missing from `/cluster/ script in k/k repository. Do we need it here?